natgeo/mortar
# [SYTO-772] Tweaks to fadeout animation after design feedback

These changes are tweaks to the animation of the POD Promo component after design review

To test:

- Pull SYTO-772
- Verify unit tests pass by running npm test
- Run gulp serve
- Verify component at http://localhost:3000/components/podpromo/index.html using the configurator
- Verify component also renders as expected at http://localhost:3000/components/
